specs/docs/security/DataHubSecurity.md

```diff
@@ -0,0 +1,266 @@
+---
+tags: [security]
+---
+
+# Data Hub Security
```

Comment:
This is great information, but I'm wondering if it should go into github yet. I don't believe we have any docs like this in github; currently, I think this information should go into a design spec on our internal Wiki. That's primarily for consistency. I do think there's long-term value in exposing this value to the public, I'm just not at the moment how best to do that - i.e. what's the right mix of info in github vs in our public docs? So for now, I think it's best to put this into a "Hub Central Security" design spec page on the Wiki.